src/com/twu/biblioteca/MainMenu.java

```diff
@@ -1,40 +1,62 @@
 package com.twu.biblioteca;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Scanner;
 
 public class MainMenu {
 
-    public void welcome() {
-        System.out.println("Welcome to Biblioteca. Your one-stop-shop for great book titles in Bangalore!");
+    private final String welcome = "Welcome to Biblioteca. Your one-stop-shop for great book titles in Bangalore!";
```

Comment:
Given these things are effectively constant over the life of your program, there is usually one more modifier we give to them during declaration.  Hint: googling "Java constants" will get you close.

Reply:
Ah yes, I researched about the final modifier once I realised I would have to abstract them out after one of your first comments about thinking how to manage the updating of the strings in the future.

Reply:
Yep, `final` is the one... `public/private static final...` is the typical signature for a constant.